### DISCUSSION TITLE
show distance between two points?

### DISCUSSION THREAD

**Discussion by connorwforman** [talmolab/sleap#1173] [Ideas]
I am very new to sleap!

Is there any way to show the distance between two points/length of an edge in pixels or something like that?

Any help is appreciated.

**Comment by talmo** [comment]
Hi @connorwforman,

We do not have an easy way to do that in the GUI at the moment, but that's a terrific suggestion! I'll move this to Ideas so we can come back to it.

In the meantime, the best thing to do would be to [export your data for analysis](https://sleap.ai/tutorials/analysis.html) so you can pull out the coordinates and compute distances and angles. [See here](https://sleap.ai/notebooks/Analysis_examples.html) for some examples.

Cheers,

Talmo
